packages/modules/imodel-browser/src/containers/ITwinGrid/ITwinGrid.tsx

```diff
@@ -90,9 +105,14 @@ export const ITwinGrid = ({
   tileOverrides,
   useIndividualState,
   postProcessCallback,
+  viewMode,
 }: ITwinGridProps) => {
   const strings = _mergeStrings(
     {
+      tableColumnName: "ITwin Number",
+      tableColumnDescription: "ITwin Name",
```

Comment:
As per our documentation site, it appears that the i of iTwin is never capitalized: 
<img width="326" alt="image" src="https://github.com/iTwin/admin-components-react/assets/1904889/bd690679-bd2f-4b13-ab4a-55c26f88bac5">

```suggestion
      tableColumnName: "iTwin Number",
      tableColumnDescription: "iTwin Name",
```